### ISSUE TITLE
In flexdashboard tables strech out on whole page/cell

### ISSUE THREAD

**Issue by rdatasculptor** [glin/reactable#98]
I am not sure if this is a something that has to be solved in `reactable `(that's my best guess) or in `flexdashboard `but I am struggling with alignments issues in reactables tables placed in flexdashboards. 

Please take a look [here](https://stackoverflow.com/questions/64908057/reactable-table-with-nested-tables-stretches-vertically-out-on-whole-column-in-f). I had no intention to cross post this issue, I just don;t know where to place it. If it has to stay on SO I will be happy to close this issue on github here.

many thanks in advance!

**Comment by rdatasculptor** [comment]
I think the main problem (for me) is that the nested table is vertically aligned to the bottom. Is this intended? And/or is there a way to prevent this behaviour to occur? [Here's a screenshot](https://i.stack.imgur.com/fOSKe.png) that shows what I mean with the alignment.

**Comment by rdatasculptor** [comment]
Thank you for your quick response!

I can show you what I mean with one your examples:

```
df <- data.frame(
  year = c(
    rep(2016, 25),
    rep(2017, 25),
    rep(2018, 25),
    rep(2019, 25)
  ),
  x = rnorm(100),
  y = rnorm(100),
  grp = c(
    rep("A", 50),
    rep("B", 50)
  )
) 
df$year <- factor(df$year, levels = c("2019","2018","2016","2017"))

df %>%
  group_by(year) %>% 
  e_charts(x, timeline = TRUE %>% 
  e_scatter(y, symbol_size = 5)

df %>%
  group_by(year) %>% 
  e_charts(x, timeline = TRUE, reorder = FALSE) %>% 
  e_scatter(y, symbol_size = 5)
```
Both plots will give a timline orderd in alphabetical/numerical order.

**Comment by glin** [comment]
Hi, try installing the latest version. This should've been fixed in version 0.2.1:
> * Table rows no longer stretch to fill the height of the container ([#69](https://github.com/glin/reactable/issues/69)).
> To make rows stretch again, use a theme like `reactableTheme(tableBodyStyle = list(flex = "auto"))`.

This is what it looks like for me:
````r
---
title: "Stretched out nested reactables"
output: flexdashboard::flex_dashboard
---

Column
-------------------------------------

### Example nested table 1

```{r}
library(reactable)
library(dplyr)

data <- MASS::Cars93[18:47, ] %>%
  mutate(ID = as.character(18:47), Date = seq(as.Date("2019-01-01"), by = "day", length.out = 30)) %>%
  select(ID, Date, Manufacturer, Model, Type, Price) %>% 
  filter(Manufacturer %in% c("Chevrolet","Chrysler"))

sales_by_mfr <- group_by(data, Manufacturer) %>%
  summarize(Quantity = n(), Sales = sum(Price))

reactable(
  sales_by_mfr,
  details = function(index) {
    sales <- filter(data, Manufacturer == sales_by_mfr$Manufacturer[index]) %>% select(-Manufacturer)
    tbl <- reactable(sales, outlined = TRUE, highlight = TRUE, fullWidth = FALSE)
    htmltools::div(style = list(margin = "12px 45px"), tbl)
  },
  onClick = "expand",
  rowStyle = list(cursor = "pointer")
)

```

Column
-------------------------------------

### Example nested table 2

```{r}
reactable(
  sales_by_mfr,
  rowStyle = list(cursor = "pointer")
)
```   
````
![flexdashboard output](https://user-images.githubusercontent.com/20401866/99884111-361e4c00-2bf1-11eb-8552-18572a34a16a.png)

Alternatively, you could prevent stretching in older versions by adding this to the table: `style = list(height = "auto")`. flexdashboard sets 100% height on all HTML widgets, but you can override that within the table:
```r
reactable(
  sales_by_mfr,
  details = function(index) {
    sales <- filter(data, Manufacturer == sales_by_mfr$Manufacturer[index]) %>% select(-Manufacturer)
    tbl <- reactable(sales, outlined = TRUE, highlight = TRUE, fullWidth = FALSE)
    htmltools::div(style = list(margin = "12px 45px"), tbl)
  },
  onClick = "expand",
  rowStyle = list(cursor = "pointer"),
  style = list(height = "auto")
)
```

**Comment by rdatasculptor** [comment]
yes, that's it! thank you for you quick, kind and educational response! updating to the newest version did the job.
